versions/1.0.0.md

```diff
@@ -238,14 +238,14 @@ version: 1.0.1
 
 Describes a source description (such as an OpenAPI description) that will be referenced by one or more workflows described within a Workflows Description.
 
-An object storing a map between named description keys and location URLs to the source descriptions (such as an OpenAPI description) this workflow SHALL apply to. Each source location `string` MUST be in the form of a [URL](https://url.spec.whatwg.org/). MAY be defined using relative references as defined by [RFC3986](https://spec.openapis.org/oas/latest.html#bib-RFC3986).
+An object storing a map between named description keys and location URLs to the source descriptions (such as an OpenAPI description) this workflow SHALL apply to. Each source location `string` MUST be in the form of a URL and MAY be defined using relative references as defined by [RFC3986](https://spec.openapis.org/oas/latest.html#bib-RFC3986).

```

Comment:
IMO we should reference RFC3986 Section 4.2 directly for relative references (https://datatracker.ietf.org/doc/html/rfc3986#section-4.2) rather than via the OAS latest reference table.

Reply:
There two places where this can be resolved